protx/utils/analytics.py

```diff
@@ -91,6 +91,10 @@ def get_distribution_prediction_plot(data, geoid=None):
             fig.add_vline(x=match.iloc[0]['pred_per_100k'],
                           line_width=3,
                           line_color="red")
+            if match.iloc[0]['pred_per_100k'] < 0:
```

Comment:
what about just changing either updating value of `match.iloc[0]['pred_per_100k']` prior to line 91.  or using python's `max` directory in line 91 (like `x=max(match.iloc[0]['pred_per_100k'], 0)`)